pkg/helmify/values.go

```diff
@@ -32,6 +33,11 @@ func (v *Values) Add(value interface{}, name ...string) (string, error) {
 	if isString {
 		return "{{ .Values." + strings.Join(name, ".") + " | quote }}", nil
 	}
+	_, isSlice := value.([]interface{})
+	if isSlice {
+		spaces := strconv.Itoa(len(name) * 2)
+		return "{{ toYaml .Values." + strings.Join(name, ".") + " | nindent " + spaces + " }}", nil
```

Comment:
Do we need `nindent` here? And if so, should it be fixed?

Can you please add a list value to one of ConfigMaps in https://github.com/arttor/helmify/blob/main/test_data/sample-app.yaml and regenerate examples in  https://github.com/arttor/helmify/tree/main/examples  in your PR?

Reply:
`nindent` is almost identical to` indent`, but begins a new line. It just makes the files more readable.
We want to dynamically generate the indentation (number of spaces) as it depends on how deeply the nested structure is rather than fixing it.

I have added a list of values to couple of configmaps to test out different scenarios with the configmap data field